### DISCUSSION TITLE
What's to be done when there is a typo in the English translation keys?

### DISCUSSION THREAD

**Discussion by ordago** [Laravel-Lang/common#33] [Q&A] [answered]
https://github.com/Laravel-Lang/lang/blob/e8eda353258c725ba14ced355e91089435ac4c18/locales/af/af.json#L36

```diff
-    "An error occured while uploading the file.": "'n fout het voorgekom met die oplaai van die lêer.",
+    "An error occurred while uploading the file.": "'n fout het voorgekom met die oplaai van die lêer.",
```

https://www.yourdictionary.com/occured

**Comment by luisprmat** [comment, marked as answer]
In theory this translation belongs to [Laravel Nova translations](https://github.com/Laravel-Lang/lang/blob/master/source/packages/nova.json#L22), but since this is a paid package I do not have the original translations to be able to corroborate.

If you use [Laravel Nova](https://nova.laravel.com) and you are sure that translation is different in that package you can formulate a PR to fix it.

**Reply by ordago** [reply]
Thanks for the info. I opened an issue over there.
